code/modules/reagents/reagent_containers/medspray.dm

```diff
@@ -32,11 +32,22 @@
 		amount_per_transfer_from_this = initial(amount_per_transfer_from_this)
 	to_chat(user, "<span class='notice'>You will now apply the medspray's contents in [squirt_mode ? "short bursts":"extended sprays"]. You'll now use [amount_per_transfer_from_this] units per use.</span>")
 
-/obj/item/reagent_containers/medspray/attack(mob/M, mob/user, def_zone)
+/obj/item/reagent_containers/medspray/attack(mob/living/carbon/M, mob/user, def_zone)
```

Comment:
Unrelated changes

Reply:
Related to patches caring about which body part you're targeting to do their effects.